### PR TITLE
Update command line docs

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Update command line docs

## Motivation and Context
After discussion with @abperiasamy review on changes to our docs

## Regression
No

## How Has This Been Tested?
Just compile and test `minio server` and `minio gateway` command

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#57138dbe-839e-43cd-8d3e-a3be04a4e81b) |
| mint-xl.sh | :x: [more...](#0c6fab58-948c-47d5-8067-b23bafda9649) |
| mint-gateway-nas.sh | :x: [more...](#cbb73e4b-ac63-4b08-81da-d2ef3ddf598b) |
| mint-large-bucket.sh | :x: [more...](#16431635-731b-4e8d-b8fb-3670ce9df6ad) |
| mint-compression-fs.sh | :x: [more...](#9a75b3b0-3808-4288-ac8e-82043fb2d218) |
| mint-worm.sh | :x: [more...](#12be45ac-4bea-459f-bf2f-89611d63f052) |
| mint-fs.sh | :x: [more...](#12400edc-c039-44d9-9885-0ee4a6e483b9) |
| mint-dist-xl.sh | :x: [more...](#18f4d877-3442-4682-a088-fcc66712b4b9) |
<a name="16431635-731b-4e8d-b8fb-3670ce9df6ad"></a>
### 6839-c95992b/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:32726
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 839c2cfbae2a:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 14 seconds
{
  "name": "awscli",
  "duration": 13199,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))"
}

Executed 0 out of 13 tests successfully.
```
<a name="12400edc-c039-44d9-9885-0ee4a6e483b9"></a>
### 6839-c95992b/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:32736
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 57b594fb898b:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 17 seconds
{
  "name": "awscli",
  "duration": 12061,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))"
}

Executed 0 out of 13 tests successfully.
```
<a name="18f4d877-3442-4682-a088-fcc66712b4b9"></a>
### 6839-c95992b/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:30806
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp bd83aff4a870:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 15 seconds
{
  "name": "awscli",
  "duration": 14310,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))"
}

Executed 0 out of 13 tests successfully.
```
<a name="12be45ac-4bea-459f-bf2f-89611d63f052"></a>
### 6839-c95992b/mint-worm.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:31143
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            worm
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 9b5a2b685fa0:/mint/log /tmp/mint-logs'

(1/1) Running worm tests ... FAILED in 4 seconds
{
  "alert": "",
  "args": {
    "bucketName": "bucket-3uxe6lk10fxvtimceuxbov2fxkfbv4gdwyf04ht44adi4j8oa5da4ns",
    "expiry": 60000000000,
    "objectName": "testObject"
  },
  "duration": 4418,
  "error": "RequestError: send request failed\ncaused by: Put http://minikube:31143/bucket-3uxe6lk10fxvtimceuxbov2fxkfbv4gdwyf04ht44adi4j8oa5da4ns: dial tcp 192.168.122.64:31143: connect: connection refused",
  "function": "PutAndDelete",
  "message": "WORM_MODE ON  - CreateBucket Failed",
  "name": "test worm mode",
  "status": "FAIL"
}

Executed 0 out of 1 tests successfully.
```
<a name="cbb73e4b-ac63-4b08-81da-d2ef3ddf598b"></a>
### 6839-c95992b/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:32644
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4101d4ae73a5:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 22 seconds
{
  "name": "awscli",
  "duration": 16550,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))"
}

Executed 0 out of 13 tests successfully.
```
<a name="0c6fab58-948c-47d5-8067-b23bafda9649"></a>
### 6839-c95992b/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:31010
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4192afebd95c:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 15 seconds
{
  "name": "awscli",
  "duration": 9739,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))"
}

Executed 0 out of 13 tests successfully.
```
<a name="57138dbe-839e-43cd-8d3e-a3be04a4e81b"></a>
### 6839-c95992b/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:30790
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp ee497c7eedf7:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 20 seconds
{
  "name": "awscli",
  "duration": 14517,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))"
}

Executed 0 out of 13 tests successfully.
```
<a name="9a75b3b0-3808-4288-ac8e-82043fb2d218"></a>
### 6839-c95992b/mint-compression-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:31910
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp f1a83b3378dc:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 24 seconds
{
  "name": "awscli",
  "duration": 18150,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "('Connection aborted.', ConnectionRefusedError(111, 'Connection refused'))"
}

Executed 0 out of 13 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Review by ebozduman — DISMISSED** [review]
LGTM

File: cmd/server-main.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ EXAMPLES:
   5. Start distributed minio server on an 8 node setup with 8 drives each. Run following command on all the 8 nodes.
```
Comment (eco-minio) [inline cmd/server-main.go]: Need to change this to 32 / 32 to coincide with new example

**Review by eco-minio — CHANGES_REQUESTED** [review]

File: cmd/server-main.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/server-main.go]: Done fixed

**Review by ebozduman — APPROVED** [review]
LGTM

**Review by eco-minio — APPROVED** [review]
LGTM

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Pull request merged**